### ISSUE TITLE
[CLI]: AttributeError: 'MailboxHandle' object has no attribute 'release'

### ISSUE THREAD

**Issue by matteo-bastico** [wandb/wandb#4779]
### Describe the bug

I run my experiment on a slurm cluster in which the computing nodes do not have internet access and therefore I am using the offline mode. After updating to version 0.13.8, I get `AttributeError: 'MailboxHandle' object has no attribute 'release'` when I init the session. Downgrading to version 0.13.7 solves the problem.

```
import wandb

if __name__ == "__main__":
   wandb.init(mode='offline')
```

```
+ srun python -u test.py
Traceback (most recent call last):
  File "/linkhome/rech/gendzm01/ujn44cd/.local/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1108, in init
    run = wi.init()
  File "/linkhome/rech/gendzm01/ujn44cd/.local/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 772, in init
    run_start_handle.release()
AttributeError: 'MailboxHandle' object has no attribute 'release'
wandb: ERROR Abnormal program exit
Traceback (most recent call last):
  File "/linkhome/rech/gendzm01/ujn44cd/.local/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1108, in init
    run = wi.init()
  File "/linkhome/rech/gendzm01/ujn44cd/.local/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 772, in init
    run_start_handle.release()
AttributeError: 'MailboxHandle' object has no attribute 'release'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/gpfsdswork/projects/rech/mwh/ujn44cd/wandb/test.py", line 5, in <module>
    wandb.init(mode='offline')
  File "/linkhome/rech/gendzm01/ujn44cd/.local/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1145, in init
    raise Exception("problem") from error_seen
Exception: problem
wandb: Waiting for W&B process to finish... (failed 1).
wandb: You can sync this run to the cloud by running:
wandb: wandb sync /gpfsdswork/projects/rech/mwh/ujn44cd/wandb/wandb/offline-run-20230112_140409-esm5azu6
wandb: Find logs at: ./wandb/offline-run-20230112_140409-esm5azu6/logs
srun: error: jean-zay-ia826: task 0: Exited with exit code 1
srun: launch/slurm: _step_signal: Terminating StepId=413943.0
```

### Environment

WandB version: 0.13.8

OS: Red Hat Enterprise Linux 8.4 (Ootpa)

Python version: 3.10.8

Versions of relevant libraries:

**Comment by Alejo-Huawei** [comment]
> Leslie commented: Hi, thank you for writing in! Can you give me your debug-internal.log found in your wandb run directory please? Can you also double check that you had input your API key properly?

Hi. I just got the same error recently. This is the debug-internal.log file:
[debug-internal.log](https://github.com/wandb/wandb/files/10459158/debug-internal.log)

I introduced the key again and it still not working.

**Comment by kptkin** [comment]
Hi @Alejo-Huawei,  This issue should've been resolved in 0.13.9, which version are you using (the problematic version is 0.13.8)?

**Comment by kptkin** [comment]
Closing the issue, if you still see the issue please feel free to re-open

**Comment by Alejo-Huawei** [comment]
Thank

> Hi @Alejo-Huawei, This issue should've been resolved in 0.13.9, which version are you using (the problematic version is 0.13.8)?

Thank you kptkin, I updated it and now it's working.
